src/lib/eslint/BaseEslintEngine.ts

```diff
@@ -243,8 +247,11 @@ export abstract class BaseEslintEngine implements RuleEngine {
 		let ruleCount = 0;
 
 		for (const rule of rules) {
+			if (!this.strategy.isRuleKeySupported(rule.name)) {
```

Comment:
Previously these rules were filtered out when running, but they were still listed in the catalog.